org.gaphor.Gaphor.appdata.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>org.gaphor.Gaphor.desktop</id>
```

Comment:
```suggestion
  <id>org.gaphor.Gaphor</id>
```
and add this tag `<launchable type="desktop-id">org.gaphor.Gaphor.desktop</launchable>`

Reply:
Updated upstream in Gaphor, thanks!